src/apiCalls.js

```diff
@@ -0,0 +1,11 @@
+const baseUrl = 'https://rancid-tomatillos.herokuapp.com/api/v2'
+
+export const getMovies = () => {
+  return fetch(baseUrl + '/movies')
+  .then(res => res.json())
+} 
+
+export const getSingleMovie = (id) => {
+  return fetch(baseUrl + `/movies${id}`)
+  .then(res => res.json())
+}
```

Comment:
These look pretty similar - can you think of any way to dry this up and just have one fetch call that can perform both fetches?